plugins/out_stackdriver/stackdriver.c

```diff
@@ -907,6 +925,13 @@ static int stackdriver_format(struct flb_config *config,
     int operation_extracted = FLB_FALSE;
     int operation_extra_size = 0;
 
+    /* Parameters for sourceLocation */
+    flb_sds_t source_location_file;
+    int64_t source_location_line = 0;
```

Comment:
nit: match the order of declaration and assignment below to make them consistent; the order here is different from the order in https://github.com/fluent/fluent-bit/pull/2400/files#diff-0780c5bd4aa79e6c53b446ed0f6a6914R1188-R1191

Reply:
Updated.